src/style.css

```diff
@@ -0,0 +1,117 @@
+/* #region reset styles */
+body {
+  margin: 0;
+  font-family: Roboto, Arial, serif;
+}
+
+h3,
+p {
+  margin: 0;
+}
```

Comment:
```suggestion
body,
h3,
p {
  margin: 0;
  /*  font-family: Roboto, Arial, serif;  */
}

```

Reply:
In my opinion,   put     font-family: Roboto, Arial, serif;      in    /* #region reset styles */    not good idea, because it is RESET AREA.